index.html

```diff
@@ -0,0 +1,220 @@
+
+    <section>
+      <h2>Technology</h2>
+      <ul>
+        <li>
+          <img src="./icons/Frame3.svg" alt="Cloud Storage">
+          <h3>Cloud Storage</h3>
+          <p>Access your account from anywhere in the world on any device</p>
+        </li>
+        <li>
+          <img src="./icons/Frame4.svg" alt="Secure">
+          <h3>Secure</h3>
+          <p>All your information is stored on secure cloud servers</p>
+        </li>
+        <li>
+          <img src="./icons/Frame6.svg" alt="PDF Download">
+          <h3>PDF Download</h3>
+          <p>Download any of your reports in PDF format</p>
+        </li>
+        <li>
+          <img src="./icons/Frame5.svg" alt="CSV Download">
+          <h3>CSV Download</h3>
+          <p>All your information is stored on secure cloud servers</p>
+        </li>
+      </ul>
+    </section>
+
+    <section>
+      <h2>Stay focused on saving money</h2>
+      <p>It is important to stay focused on saving money in any way you can. We help you monitor your spending
+        habits
+        so you can easily spot and cut any unnecessary expenses. Simply join today to get started!
+      </p>
+      <ul>
+        <li>
+          <img src="./icons/ion-logo-google-playstore.svg" alt="google playstore">
+        </li>
+        <li>
+          <img src="./icons/ion-logo-apple-appstore.svg" alt="apple appstore">
+        </li>
+        <li>
+          <img src="./icons/simple-icons_windows.svg" alt="windows">
+        </li>
+      </ul>
+      <iframe
+        src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2748.444850550622!2d30.747026515109138!3d46.45969587448731!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40c633d86691cbe7%3A0xa529f2c900d55963!2z0JzRg9C30LXQuSDQmNC90YLQtdGA0LXRgdC90L7QuSDQvdCw0YPQutC4!5e0!3m2!1sru!2sua!4v1613304636744!5m2!1sru!2sua">
+      </iframe>
+    </section>
+
+    <section>
+      <h2>Contact</h2>
+      <p>Questions or concerns? Just fill out the form below and our support team will get back to you within 24
+        hours. But I must explain to you how all this mistaken idea of denouncing pleasure and praising pain was
+        born and I will give you a complete account of the system, and expound the actual teachings of the great
+        explorer of the truth, the master-builder of human happiness.
+      </p>
+      <form>
+        <label>
+          <input name="user_first_name" type="text" placeholder="First Name">
+        </label>
+        <label>
+          <input name="user_last_name" type="text" placeholder="Last Name">
+        </label>
+        <label>
+          <input name="user_phone" type="tel" placeholder="Phone Number">
+        </label>
+        <label>
+          <select name="user_service">
+            <option selected>Select Service</option>
+            <option value="1">Service 1</option>
+            <option value="2">Service 2</option>
+            <option value="3">Service 3</option>
+          </select>
+        </label>
```

Comment:
There is no need to add `label` as a wrapper for `input` in this case. Label has no text inside, so it is useless. It will be better to add aria-label for each form element to improve a11y.